publish.sh

```diff
@@ -15,13 +15,13 @@ for PKG in */pyproject.toml; do
     cp pyproject.toml pyproject.toml.bak
     PKG_VERSION="$(dunamai from git)"
 
+    echo "__version__ = '$PKG_VERSION'" > ./"$PKG"/version.py
```

Comment:
Needs to be `./"$PKG"/version.py`, because we already pushd-ed into `$PKG`

Reply:
Thank you, great catch!